fe/src/main/java/org/apache/doris/persist/AlterViewInfo.java

```diff
@@ -35,15 +38,18 @@
     private String inlineViewDef;
     @SerializedName(value = "sqlMode")
     private long sqlMode;
+    private List<Column> newFullSchema;
```

Comment:
You can also add @SerializedName to `newFullSchema`

Reply:
done